turbo.json

```diff
@@ -96,8 +96,21 @@
     },
     "codegen": {
       "cache": true,
-      "outputs": ["src/api/**/*.ts", "!src/api/client.ts", "!src/api/client.test.ts"],
+      "outputs": [
+        "src/api/**/*.ts",
+        "!src/api/client.ts",
+        "!src/api/client.test.ts"
+      ],
```

Comment:
If you don't mind, it's better not to include changes/refactoring not relevant to the task in the PR. Even a formatting change already requires the reviewer's attention.

Reply:
Sure, let's cover CI stuff in a next PR then